packages/nuxt/src/app/entry.ts

```diff
@@ -76,6 +85,8 @@ if (import.meta.client) {
       nuxt.payload.error = (nuxt.payload.error || err) as any
     }
 
+    vueApp.config.errorHandler = prevVueErrorHandler
```

Comment:
Reset back to undefined. Or should we actually always provide the handler with a better experience?

Reply:
Interesting question. I'd be up for always providing an error handler, but maybe we can get some thoughts from rest of team as well?